internals/overlord/state/change.go

```diff
@@ -157,7 +181,6 @@ type marshalledChange struct {
 	Clean   bool                        `json:"clean,omitempty"`
 	Data    map[string]*json.RawMessage `json:"data,omitempty"`
 	TaskIDs []string                    `json:"task-ids,omitempty"`
-	Lanes   int                         `json:"lanes,omitempty"`
```

Comment:
Is this safe for us to just remove? What will be the impact for current systems ?